package.json

```diff
@@ -47,6 +47,7 @@
     "jose": "npm:jose-node-cjs-runtime@^3.11.3",
     "js-yaml": "^4.0.0",
     "lodash.partition": "^4.6.0",
+    "luxon": "^3.0.4",
```

Comment:
Can this be in `devDependencies` since scripts/remind-to-promote.js is not actually used by the app?

Reply:
Ah, yes.

Reply:
Fixed in repush.

Reply:
Lint is now failing:

```
> eslint --ext .js,.jsx .


/home/runner/work/nextstrain.org/nextstrain.org/scripts/remind-to-promote.js
Error:   3:1  error  'luxon' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
```

This is tricky because we have multiple projects in this repo and ESLint is using the same rules for all projects. Yet, some rules such as `import/no-extraneous-dependencies` are project-specific.

One solution would be to limit ESLint to just `src/`, but it's nice to have some form of linting on other JS files.

Another solution, which retains global linting, would be to create a new `scripts/package.json`. This rule would then search there based on https://github.com/import-js/eslint-plugin-import/issues/1174.

Reply:
Oy. Fixed with [34df2bc](https://github.com/nextstrain/nextstrain.org/pull/615/commits/34df2bc3a9ece6551de451b200c848aeb1d305c9).